content/en/about/contributing/syntax/_index.md

```diff
@@ -58,7 +94,8 @@ The following code:
 
 Will render as:
 
-{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
+{{% pageinfo %}} This is a
+placeholder. {{% /pageinfo %}}
```

Comment:
```suggestion
{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
```

Reply:
This render weird in some places. I discovered that adding a line break will render it aligned in the middle vertically.